data/linux-bridge/002-linux-bridge.yaml

```
@@ -32,14 +32,15 @@ spec:
             - /bin/bash
             - -c
             - |
-              cp -rf /usr/src/containernetworking/plugins/bin/*bridge /opt/cni/bin/
-              cp -rf /usr/src/containernetworking/plugins/bin/*tuning /opt/cni/bin/
+              echo 'Installing bridge and tuning CNIs'
+              cp -f /usr/src/containernetworking/plugins/bin/*bridge /opt/cni/bin/
+              cp -f /usr/src/containernetworking/plugins/bin/*tuning /opt/cni/bin/
               # Some projects (e.g. openshift/console) use cnv- prefix to distinguish between
               # binaries shipped by OpenShift and those shipped by KubeVirt (D/S matters).
               # Following two lines make sure we will provide both names when needed.
-              find /opt/cni/bin/cnv-bridge || ln -s /opt/cni/bin/bridge /opt/cni/bin/cnv-bridge
-              find /opt/cni/bin/cnv-tuning || ln -s /opt/cni/bin/tuning /opt/cni/bin/cnv-tuning
-              echo "Entering sleep... (success)"
+              find /opt/cni/bin/cnv-bridge &>/dev/null || ln -s /opt/cni/bin/bridge {{ .CNIBinDir }}/cnv-bridge
+              find /opt/cni/bin/cnv-tuning &>/dev/null || ln -s /opt/cni/bin/tuning {{ .CNIBinDir }}/cnv-tuning
```

Let's just re-create the link if it exists.
```suggestion
              ln -fs /opt/cni/bin/bridge {{ .CNIBinDir }}/cnv-bridge
              ln -fs /opt/cni/bin/tuning {{ .CNIBinDir }}/cnv-tuning
```

But current code is wrong since we are checking the link it has to be 
```suggestion
              find {{ .CNIBinDir }}/cnv-bridge &>/dev/null || ln -s /opt/cni/bin/bridge {{ .CNIBinDir }}/cnv-bridge
              find {{ .CNIBinDir }}/cnv-tuning &>/dev/null || ln -s /opt/cni/bin/tuning {{ .CNIBinDir }}/cnv-tuning
```

---

Merged this too quickly. We cannot recreate it. The issue is that on D/S, the binary name is cnv-bridge, on U/S it is bridge. In order to make the cnv-bridge binary available on D/S, we just dump it on the host. In order to make it available on U/S (for UI testing), we make a symlink to bridge called cnv-bridge